### DISCUSSION TITLE
MIT 计算机科学与工程 | hackway.org

### DISCUSSION THREAD

**Discussion by giscus** [lidongyx/hackwaydoc#17] [Announcements]
# MIT 计算机科学与工程 | hackway.org

麻省理工MIT既是世界顶级名校，在计算机方面也是四大CS名校之一。它的计算机专业隶属于电子工程与计算机学院Electrical Engineering and Computer Science。由于MIT在开放课程上一直做得比较优秀，Syllabus课程表、Readings教材与参考资料、Slides课件、Code代码、video课程视频以及Lab实验课等等都十分齐全，因此十分推荐大家学习MIT CS的课程。

https://hackway.org/docs/cs/path/stanford

**Comment by lidongyx** [comment]
斯坦福大学官方也提供了一个课程学习路线图，下个版本更新的时候会补上

**Reply by fenglei666** [reply]
编程基础看不了了

**Comment by fenglei666** [comment]
编程基础网页看不了了，能修复一下吗
